instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/properties/SpringConfigProperties.java

```diff
@@ -117,29 +118,45 @@ public Duration getDuration(String name) {
   @SuppressWarnings("unchecked")
   @Override
   public Map<String, String> getMap(String name) {
+    Map<String, String> fallbackMap = fallback.getMap(name);
     // maps from config properties are not supported by Environment, so we have to fake it
     switch (name) {
       case "otel.resource.attributes":
-        return resourceProperties.getAttributes();
+        return mergeWithFallback(resourceProperties.getAttributes(), fallbackMap);
```

Comment:
` fallbackMap` is a confusing anfd vague for me, `propertiesFromOTelSdk`?

Reply:
@jeanbisutti please check if it's clear now